### ISSUE TITLE
Bugsnag optional for whitelabel builds

### ISSUE THREAD

**Issue by newmanw** [RocketChat/Rocket.Chat.ReactNative#3043]
### Description:

This is related to #2971, but for different reasons.  I support a number of government customers that would like to bring Rocket.Chat into their enterprise.  One reason why applications like Rocket.Chat are invaluable to the government is the ability to host these systems internally. Restrictions when running on a government system are high, one of these being communication with external 3rd party systems. Unfortunately running with bugsnag enabled is a non starter.  Would it be possible to look into adding an option to disable bugsnag when building the whitelabel apps? 

### Environment Information:

- Rocket.Chat Server Version: 13.12.1
- Rocket.Chat App Version: 4.15.0
- Device Name: Any
- OS Version: Any

**Comment by vplb** [comment]
We have similar issues and requirements. It's absolutely not ideal but we are testing jamming the data by DNS. 

A note, we have tried to change the Bugsnag key to something that will fail, and when it failed it actually ended up taking down the server or causing other technical problems. 

I would argue that it should be addressed in all builds: *any* communication involving data collection, on the back end with no user visibility or interaction especially,  _**should be opt-in**_, and very obvious on the installer.

**Comment by unsignedString** [comment]
I totally agree. Such a feature should be opt-in or easily deactivatable by the app user at least - but I as an admin would prefer to have the opportunity to exclude it from compiling to be sure it is not part of the app.

**Comment by Rohit3523** [comment]
For this, I can include a guide how to remove BugSnag from the code level :)
